### PR TITLE
more build fixes

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,6 @@ script:
 deploy:
   - provider: script
     skip_cleanup: true
-    script: workshop/deploy_rollup.sh
+    script: echo workshop/deploy_rollup.sh
```
Comment (AnthonyAmanse) [inline .travis.yml]: Why is it `echo` here?

File: workshop/bx_login.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ if [ -z $CF_SPACE ]; then
 fi
 
 
-if [ -z "$BLUEMIX_API_KEY" || [ -z "$BLUEMIX_NAMESPACE" ]; then
+if [ -z "$BLUEMIX_API_KEY" ] || [ -z "$BLUEMIX_NAMESPACE" ]; then
```
Comment (AnthonyAmanse) [inline workshop/bx_login.sh]: Don't forget to set the environment variables here https://travis-ci.org/IBM/kube101/settings

File: .travis.yml (hunk 1 above)
Comment (AnthonyAmanse) [reply .travis.yml]: You can create one here https://console.bluemix.net/iam/#/apikeys (the account should be the one who has access to the `CLUSTER_NAME` and the namespace)

For the namespace,
https://github.com/IBM/kube101/blob/2633eed2a89b90c80387ec3001caba9820b85d2c/workshop/deploy.sh#L76

I'm guessing its `contbot`?
https://github.com/IBM/kube101/blob/2633eed2a89b90c80387ec3001caba9820b85d2c/workshop/deploy.sh#L49

Comment (duglin) [reply .travis.yml]: yea, I'm sure its possible, but do we have an existing cluster/org/space/.... setup for testing already?

**Comment by duglin** [comment]
Disabling until we know if we have an account to use.

**Pull request merged**